README.md

```diff
@@ -62,7 +62,7 @@ The technical documents are intended for a technical audience and represent the
 * [Corona-Warn-App Verification Portal Server Software Design](https://github.com/corona-warn-app/cwa-verification-portal/blob/master/docs/architecture-overview.md)
 * [Corona-Warn-App Test Result Server Software Design](https://github.com/corona-warn-app/cwa-testresult-server/blob/master/docs/architecture-overview.md)
 * [Corona-Warn-App Mobile Client (Android) Architecture](https://github.com/corona-warn-app/cwa-app-android/blob/master/docs/architecture-overview.md)
-* [Corona-Warn-App Mobile Client (iOS) Architecture](https://github.com/corona-warn-app/cwa-app-ios/blob/development/docs/architecture-overview.md)
+* [Corona-Warn-App Mobile Client (iOS) Architecture](https://github.com/corona-warn-app/cwa-app-ios/blob/develop/docs/architecture-overview.md)
```

Comment:
they have both `develop` and `master`. I guess master is more accurate no? at least that's the branch they use for the android docs.

Reply:
I primarily guided myself by the previous branch, but your point sounds reasonable too 👍. `master` seems more suitable logically and looks more consistent compared to the other docs while `develop` is one commit ahead. Maybe @SebastianWolf-SAP can comment on this, as he originally added the links to the architecture docs 😊

Reply:
develop is fine, as it is the main branch. Would be great if Github allowed links to the default branch, but for now let's just remove the 404 ... thanks a lot!